packages/types/src/tokens.ts

```diff
@@ -3,9 +3,11 @@ export type ValidateTokenArguments = {
 };
 
 export type ValidateTokenAPIResponse = {
-  isValid: boolean;
+  isValid: TokenValidity;
 };
 
+type TokenValidity = boolean;
```

Comment:
What do you think of exposing the validity directly? 
```ts
// new
const [isValid] = await slashauthClient.validateToken(token);

// old
const [res] = await slashauthClient.validateToken(token);
res.isValid
const [{ isValid }] = await slashauthClient.validateToken(token);
```

Reply:
I think the issue with this and hasRole is that I can't just do:
```
if (await slashauthClient.validateToken(token)) {
// logic
}
```

for all the non-bools, this works nicely

Reply:
Yeah, that is one of the downsides of returning an array... what do you suggest?